src/index.html

```diff
@@ -1,12 +1,311 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="navigation">
+      <div class="navigation__container">
+        <a href="#home" class="navigation__logo">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="navigation__section">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a href="#" class="navigation__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navigation__item" data-qa="nav-hover">
+            <a href="#" class="navigation__link">
+              Laptops&nbsp;&&nbsp;computers
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card">
```

Comment:
it's better not set such classes for main. 
You can create an own wrapper for your card and name it as "cards__container"